src/com/facebook/buck/go/GoSymlinkTree.java

```diff
@@ -39,10 +40,9 @@
 
   protected GoSymlinkTree(
```

Comment:
Obviously, a but outside the scope of this diff, but why can't we just reuse `SymlinkTree` here?  The only different seems that we want to create the links via walking the deps, but this looks like it could easily be done in a method that does the walk to build up the link map, then just news up a `SymlinkTree`.